examples/controls_example/public/react_controls/data_controls/range_slider/has_no_results.ts

```diff
@@ -92,7 +93,7 @@ async function hasNoResults({
   // "has no results" or "has results" vs the actual count
   searchSource.setField('trackTotalHits', 1);
 
-  const allFilters = unifiedSearchFilters ? unifiedSearchFilters : [];
+  const allFilters = filters ? [...filters] : [];
```

Comment:
Why the spread here, if `filters` is of type `Filter[]`? Can `filters` sometimes just be `Filter`? Not sure we need to clone the filters here if the type is accurate

Reply:
spreading to avoid mutating the passed in parameter